src/drivers/imx/ipc.c

```diff
@@ -60,7 +60,9 @@ static void irq_handler(void *arg)
 		imx_mu_write(IMX_MU_xSR_GIPn(IMX_MU_VERSION, 1),
 			     IMX_MU_xSR(IMX_MU_VERSION, IMX_MU_GSR));
 
+#ifndef CONFIG_ARM64
```

Comment:
Can you please explain why it souldn't exist interrupt_clear() for ARM64?

Reply:
Some extra clarifications here: https://github.com/thesofproject/sof/pull/7470. Please let me know if you think the commit description is not clear enough. I can try re-writing it.

Reply:
"In the case of ARM64 with GIC the interrupts are cleared after the ISR is executed (as a reference please see arm_gic_eoi() from Zephyr and arch/arm64/core/isr_wrapper.S ) so there's no need to clear them explicitly." - this is from your [comment ](https://github.com/thesofproject/sof/pull/7470/commits/0626a37ae32d36c98b7ecd8b12fa50f723a45147#r1172276504). 
This was useful for me, so it might be good to add it in the commit message.